.github/workflows/maven-checks.yml

```
@@ -46,5 +46,69 @@ jobs:
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           ./mvnw install -B -V -T 1C -DskipTests -Dmaven.javadoc.skip=true --no-transfer-progress -P ci -pl '!presto-test-coverage,!:presto-docs'
+      - name: Upload presto-server tarball
+        if: matrix.java == '8.0.442'
+        uses: actions/upload-artifact@v4
+        with:
+          name: presto-server
+          path: presto-server/target/presto-server-0.293.tar.gz
+      - name: Upload presto-cli executable
+        if: matrix.java == '8.0.442'
+        uses: actions/upload-artifact@v4
+        with:
+          name: presto-cli
+          path: presto-cli/target/presto-cli-0.293-executable.jar
       - name: Clean Maven Output
         run: ./mvnw clean -pl '!:presto-server,!:presto-cli,!presto-test-coverage'
+  
+  presto-coordinator-with-clp-connector-runtime-image:
+    name: presto-coordinator-runtime-image
+    needs: maven-checks
+    runs-on: ubuntu-22.04
+    if: ${{ always() && success() }}
+    steps:
+      - uses: "actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683"
+        with:
+          submodules: "recursive"
+
+      - name: Download presto-server
+        uses: actions/download-artifact@v4
+        with:
+          name: presto-server
+          path: ./docker
+
+      - name: Download presto-cli
+        uses: actions/download-artifact@v4
+        with:
+          name: presto-cli
+          path: ./docker
+
+      - name: "Login to image registry"
+        uses: "docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772"
+        with:
+          registry: ghcr.io
+          username: ${{github.actor}}
+          password: ${{secrets.GITHUB_TOKEN}}
+
+      - name: "Update Metadata"
+        id: "meta"
+        uses: "docker/metadata-action@902fa8ec7d6ecbf8d84d538b9b233a880e428804"
+        with:
+          images: >-
+            ghcr.io/${{github.repository}}/coordinator-with-clp-connector-runtime
+          tags: |
+            type=raw,value=centos9
```

```suggestion
            type=raw,value=dev
```

When we need to release these images, we can create a branch and change the tag to something more sensible.